apps/wallet-mobile/src/components/Modal/ModalScreen.tsx

```diff
@@ -108,7 +108,7 @@ const useStyles = () => {
     },
     sheet: {
       flex: 1,
-      backgroundColor: color.gray_c50,
+      backgroundColor: isDark ? color.gray_c50 : color.white_static,
```

Comment:
@banklesss this shouldn't happen, can you check with Design why there is no subtoken for this case?

Reply:
maybe is just legacy....

Reply:
i asked them and they want a specifically white background in light mode and that color in dark

Reply:
🤔

Reply:
i will ask them again